syz-manager/manager.go

```diff
@@ -177,6 +177,7 @@ func RunManager(cfg *mgrconfig.Config) {
 
 	mgr.preloadCorpus()
 	mgr.initHTTP() // Creates HTTP server.
+	mgr.initStats() // Initializes prometheus variables
```

Comment:
I think we init stats before html, otherwise Prometheus query arriving at a wrong time will get empty counters or an error.

Reply:
Good catch. Resolved in the revision